src/inspect.js

```diff
@@ -47,11 +48,8 @@ export function silenceError (p) {
 	p._runAction(silencer)
 }
 
-const silencer = {
-	fulfilled () {},
-	rejected: setHandled
-}
-
-function setHandled (rejected) {
-	rejected._state |= HANDLED
+const silencer = new Action(null)
```

Comment:
I somehow believed that this would cause `silencer` to have the same hidden class as other actions so it could be superior. Not sure, not tested.
Maybe I should completely drop changes to silencing until I figure out whether there need to be some larger changes.